includes/media_bynder.admin.inc

```diff
@@ -354,6 +357,16 @@ function media_bynder_generate_selected_facets($selected_facets) {
         $facets_list['#items'][] = $facet_category;
     }
 
+    $facet_remove_all = array(
+        'data' => "<span style='background:#1ca0de'>" . 'Clear facets' . "<span class='expand'><i class='fa fa-times'></i></span></span>",
```

Comment:
Also what's up with the mixing of quote styles and concetanation? Doesn't seem necessary